src/pykeen/inverse.py

```diff
@@ -61,6 +65,10 @@ def invert_(self, batch: torch.LongTensor, index: int = 1) -> torch.LongTensor:
         batch[:, index] += 1
         return batch
 
+    # docstr-coverage: inherited
+    def is_inverse(self, ids: torch.LongTensor) -> torch.BoolTensor:  # noqa: D102
```

Comment:
somewhere there is also the inverse index generator function, should these be coupled?

Reply:
This is exactly what this class is for 😉 

The full inversion combines `_map` and `invert_` defined in this concrete class, `DefaultRelationInverter` ,  in `map` in the base class, `RelationInverter`.

Reply:
This PR is mostly a fix for the prediction dataframe; there are still some inconsistencies to fix (e.g. that inverse relations are stored in a dataset instead of solely encapsulated in a model; that tf.relation_to_id contains only the relation ids *before* remapping, etc.), cf. this long-standing PR: https://github.com/pykeen/pykeen/pull/752